Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3619,7 +3619,7 @@ public void run() {
 	 * @see BranchError	
 	 */
 	public interface BranchListResponseListener {
-		public void onReceivingResponse(JSONArray list, BranchError error);
+		void onReceivingResponse(JSONArray list, BranchError error);
```

Comment:
http://docs.oracle.com/javase/specs/jls/se7/html/jls-9.html#jls-9.4

In C# the compiler will raise if you do this, and it's not only redundant, it also implies that an interface method __could__ be non-public.